src/commands/keygen.rs

```diff
@@ -101,14 +103,31 @@ impl LdnsCommand for Keygen {
                 }
 
                 Arg::Short('s') => {
-                    create_symlinks = true;
+                    #[cfg(target_family = "unix")]
+                    {
+                        create_symlinks = true;
+                    }
+                    #[cfg(not(target_family = "unix"))]
+                    return Err("symlinks not supported outside Unix platforms".into());
                 }
 
                 Arg::Short('f') => {
-                    force_symlinks = true;
+                    #[cfg(target_family = "unix")]
+                    {
+                        force_symlinks = true;
+                    }
+                    #[cfg(not(target_family = "unix"))]
+                    return Err("symlinks not supported outside Unix platforms".into());
+                }
+
+                Arg::Short('v') => {
+                    let version = clap::crate_version!();
+                    // NOTE: The outer version is the latest version of 'ldns-keygen' we are
+                    // compatible with.  This needs to be updated manually.
+                    println!("DNSSEC key generator version 1.8.4 (dnst version {version})");
+                    std::process::exit(0);
```

Comment:
Would you mind waiting with this? It's incompatible with https://github.com/NLnetLabs/dnst/pull/23.